server/src/main/java/org/opensearch/common/remote/AbstractRemoteWritableBlobEntity.java

```diff
@@ -44,6 +44,10 @@ public AbstractRemoteWritableBlobEntity(final String clusterUUID, final Compress
         this(clusterUUID, compressor, null);
     }
 
+    public AbstractRemoteWritableBlobEntity() {
+        this(null, null, null);
+    }
```

Comment:
Why do we need this constructor?

Reply:
It was used in `RemoteIndexRoutingTableDiff(InputStream inputStream)` constructor, removed it in latest revision.